docs/DEVELOPMENT.md

```diff
@@ -71,34 +74,35 @@ By default bounds to port 8080
 DEBUG="True" venv/bin/python manage.py runserver 8000
 ```
 
-* Run test suite
+* Run the test suite
 
 ```
-cd laa-fee-calculator
 venv/bin/python manage.py test
 ```
 
-or, using pyenv
+* Run an individual test
+
 ```
-cd laa-fee-calculator
-pyenv activate laa-fee-calculator
-./manage.py test
+venv/bin/python manage.py test calculator.tests.test_calculation_agfs_12
 ```
 
-run individual test
+**Alternative Python version management**
```

Comment:
I haven't tried `pyenv` but you can get the same result with;

```bash
cd laa-fee-calculator
source venv/bin/activate
```

Do you know what benefit pyenv has?

(I know you have just moved this around without changing so I'm happy to approve this as it is)

Reply:
I'm not really clear of the benfit of `pyenv` vs `venv`; I need to start on my Python L&D! My basic understanding is that they're two ways of doing the same thing and the instruction is just there to cater for devs who use different tools (similar to `rbenv` vs `rvm` etc in Ruby).

Maybe it's better to just remove it? This page is there to get the app running, not as a Python primer, so maybe it's overkill.

Reply:
I installed `pyenv` to try this and I get;

```
pyenv: no such command `activate'
```

Maybe I'm missing something in the setup. I would agree with removing it as anyone using `pyenv` would presumably not need the instructions.

Reply:
Agreed. I've removed it now 👍